tsdb/engine/tsm1/reader.go

```diff
@@ -145,11 +148,12 @@ func (b *BlockIterator) Next() bool {
 	return false
 }
 
-func (b *BlockIterator) Read() (string, int64, int64, uint32, []byte, error) {
+// Read reads information about the next block to be iterated.
+func (b *BlockIterator) Read() (key string, minTime int64, maxTime int64, checksum uint32, buf []byte, err error) {
 	if b.err != nil {
 		return "", 0, 0, 0, nil, b.err
 	}
-	checksum, buf, err := b.r.readBytes(&b.entries[0], nil)
+	checksum, buf, err = b.r.readBytes(&b.entries[0], nil)
```

Comment:
I named the return values in this method, which changed this assignment all to existing variables.